frontend/src/metabase/embedding/embedding-hub/components/EmbeddingHub.tsx

```diff
@@ -1,19 +1,22 @@
-import cx from "classnames";
 import { useMemo, useState } from "react";
-import { c, t } from "ttag";
+import { P, match } from "ts-pattern";
+import { t } from "ttag";
 
-import CS from "metabase/css/core/index.css";
+import MetabotLogo from "metabase/common/components/MetabotLogo";
 import { CreateDashboardModal } from "metabase/dashboard/containers/CreateDashboardModal";
 import { AddDataModal } from "metabase/nav/containers/MainNavbar/MainNavbarContainer/AddDataModal";
-import { Box, Icon, Menu, Text, Title } from "metabase/ui";
+import { Box, Group, Stack, Text } from "metabase/ui";
 
 import { useCompletedEmbeddingHubSteps } from "../hooks";
-import type { EmbeddingHubStepId } from "../types";
 import type { EmbeddingHubModalToTrigger } from "../types/embedding-checklist";
 import { getEmbeddingHubSteps } from "../utils";
 
-import { EmbeddingHubChecklist } from "./EmbeddingHubChecklist";
 import { EmbeddingHubXrayPickerModal } from "./EmbeddingHubXrayPickerModal";
+import {
+  type StepperCardClickAction,
+  type StepperStep,
+  StepperWithCards,
+} from "./StepperWithCards/StepperWithCards";
 
 export const EmbeddingHub = () => {
   const embeddingSteps = useMemo(() => getEmbeddingHubSteps(), []);
```

Comment:
Unrelated to this PR but what is the rationale behind the steps being returned by a function, instead of simply being a const? This adds unnecessary overhead here (and also in `embedding-hub-steps.ts`).

Reply:
@lorem--ipsum because of translations: if you make it a `const EMBEDDING_HUB_STEPS`, then the ttag translations no longer works as all translations must be called dynamically.

Reply:
That makes sense